linux-anvil-aarch64-cuda/Dockerfile

```
@@ -45,6 +45,14 @@ RUN if [ "$CENTOS_VER" -le "7" ]; then \
 # Remove preinclude system compilers
 RUN rpm -e --nodeps --verbose gcc gcc-c++
 
+# Fallback to CentOS vault for CentOS 8 support.
+RUN if [ "$CENTOS_VER" -eq "8" ]; then \
+        find /etc/yum.repos.d/ -name "CentOS-*.repo" -exec \
+             sed -i 's/mirrorlist/#mirrorlist/g;s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' {} \; && \
+        yum update -y && \
+        yum clean all; \
```

Elsewhere a custom script is being used for `yum clean all`. This is primarily needed for CentOS 6, which isn't relevant here. However it does all delete files in the cache, which would be useful here. Plus this script is being used elsewhere. So this should be added here.

```suggestion
        yum_clean_all; \
```